pkgs/development/python-modules/mediafile/default.nix

```diff
@@ -19,7 +19,7 @@ buildPythonPackage rec {
     owner = "beetbox";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-2h17FA0GTY4R+WhZiQtPFYf6gH7XLbI3aOB/nUXFtJI=";
+    hash = "sha256-5HHfG1hCIbM/QSXgB61yHNNWJTsuyAh6CQJ7SZhZuvo=";
   };
 
   nativeBuildInputs = [
```

Comment:
We can drop the six dependency

Reply:
I don't think I can remove yet; I think they only removed it as a dependency in 0.13.0.

Reply:
Ups, yeah true https://github.com/beetbox/mediafile/commit/f2db5237e435e788fc810353df8756e23e38ddd6